src/bridge/command.rs

```diff
@@ -48,26 +48,26 @@ fn build_nvim_cmd() -> TokioCommand {
 }
 
 // Creates a shell command if needed on this platform (wsl or macos)
-fn create_platform_shell_command(command: String) -> Option<StdCommand> {
+fn create_platform_shell_command(command: &str, args: Vec<&str>) -> Option<StdCommand> {
```

Comment:
Nit: If you change the signature here already, I think it's just fine to take a slice here (`&[&str]`), too.